src/main/java/com/idrsolutions/microservice/BuildVuServlet.java

```diff
@@ -202,12 +211,17 @@ protected boolean validateRequest(final HttpServletRequest request, final HttpSe
      * occurs
      */
     private static boolean convertToPDF(final File file, final Individual individual) {
-        final ProcessBuilder pb = new ProcessBuilder("soffice", "--headless", "--convert-to", "pdf", file.getName());
+        final String uuid = individual.getUuid();
+        final String uniqueLOProfile = TEMP_DIR + "LO-" + uuid;
+
+        final ProcessBuilder pb = new ProcessBuilder("/opt/libreoffice6.4/program/soffice",
```

Comment:
@leonatherton - Though I'm very happy with the change in functionality for LibreOffice, I think hard-coding a set install location and version of LibreOffice is a regression from the original process. 

Could we look into adding a way for this to be configured? Perhaps by treating the LibreOffice installation as a variable in the same way we do with INPUTPATH/OUTPUTPATH?  

Furthermore should we make the default something more generic like "/usr/bin/soffice" rather than "/opt/libreoffice6.4/program/soffice"?

Reply:
@daniel-warren well spotted, I didn't intend to alter the soffice path (that was just me testing on a local VM).

I think the other suggestion is a good idea but outside the scope of this task - would suggest adding to the ideas list on Basecamp.